lib/desired-caps.js

```diff
@@ -2,317 +2,324 @@
 const PLATFORM_NAME_IOS = 'iOS';
 const PLATFORM_NAME_TVOS = 'tvOS';
 
-const desiredCapConstraints = /** @type {const} */({
-  platformName: { // override
+const desiredCapConstraints = /** @type {const} */ ({
+  platformName: {
```

Comment:
please revert these automatic changes. They prevent seeing what's really changed. We could extract them into a separate PR if needed

Reply:
Yea... let me skip the commit hook to change this file